src/libs/fileDownload/FileUtils.js

```diff
@@ -181,7 +182,7 @@ const readFileAsync = (path, fileName, onSuccess, onFailure = () => {}) =>
                 return res.blob();
             })
             .then((blob) => {
-                const file = new File([blob], cleanFileName(fileName), {type: blob.type});
+                const file = new File([blob], cleanFileName(fileName), {type: blob.type || type});
```

Comment:
What's this change for? blob doesn't return type in updated version?

Reply:
I tested and confirmed that type isn't returned in old version as well.
So what exactly does this change fix?

Reply:
For only the files which does have `Space` in their name, as you told upload failing for files which does have space init, after some digging `type` is the issue for the files which does have the space.

Reply:
Nice! But I said it's out of scope as it already happens on main and has different root cause.
Let's just fix crash here.

Reply:
oh, ok then. Fine. Will update the code to just have the fix.